slice-record/nsir-schema.yml

```diff
@@ -41,7 +41,7 @@ properties:
     type: string
     enum:
     - INSTANTIATED
-    - NOT_INSTANTIATED
+    - TERMINATE
```

Comment:
@alemanyp I think should be TERMINATED, and the description should be updated as well

Reply:
@felipevicens Done